### ISSUE TITLE
The train dataset will be loaded too when running eval.py

### ISSUE THREAD

**Issue by wnzhyee** [MhLiao/DB#199]
Is that neccessary?

**Comment by wnzhyee** [comment]
Another question, it seems I cannot change the base learning rate by changing the yaml config file or use --lr command, only when i change the default lr setting in learning_rate.py, it can work. Is there something wrong?

**Comment by wsibo** [comment]
> Is that neccessary?

I comment out "train: data_loader" in the yaml config and it works. Seems when construct the structure from yaml, it will automatically build the dataset class and get_all_samples() in __init__() func.

**Comment by lifei1229** [comment]
Hi i also met this question, can you solve now?
